clang/lib/DPCT/AnalysisInfo.cpp

```diff
@@ -2389,7 +2389,7 @@ std::string DpctGlobalInfo::SYCLHeaderExtension = std::string();
 clang::tooling::UnifiedPath DpctGlobalInfo::CudaPath;
 std::string DpctGlobalInfo::RuleFile = std::string();
 UsmLevel DpctGlobalInfo::UsmLvl = UsmLevel::UL_None;
-BuildScriptKind DpctGlobalInfo::BuildScriptVal = BuildScriptKind::BS_None;
+unsigned DpctGlobalInfo::BuildScriptType = 0;
```

Comment:
if BuildScriptType default to zero,  then it default to (BS_CMake), then migrateCMakeScripts() will be defatult return true.  which conflict with option desc:  "By default, no build script is migrated.\n" in DPCTOptions.inc.

Reply:
Here, BuildScriptType holds the value of the selected option by user (by cl::bits option), not the BS_CMake enum value.
So,
0 => No build script migration
1 => CMake script migration
2 => Python script migration
3 => Both CMake and Python script migration